src/EventService.Business/Commands/Event/CreateEventCommand.cs

```diff
@@ -28,44 +28,73 @@ public class CreateEventCommand : ICreateEventCommand
   private readonly IAccessValidator _accessValidator;
   private readonly IResponseCreator _responseCreator;
   private readonly IHttpContextAccessor _contextAccessor;
+  private readonly IEmailService _emailService;
+  private readonly IUserService _userService;
 
   public CreateEventCommand(
     IEventRepository repository,
     ICreateEventRequestValidator validator,
     IDbEventMapper mapper,
     IAccessValidator accessValidator,
     IResponseCreator responseCreator,
-    IHttpContextAccessor contextAccessor)
+    IHttpContextAccessor contextAccessor,
+    IUserService userService,
+    IEmailService emailService)
   {
     _repository = repository;
     _validator = validator;
     _mapper = mapper;
     _accessValidator = accessValidator;
     _responseCreator = responseCreator;
     _contextAccessor = contextAccessor;
+    _userService = userService;
+    _emailService = emailService;
+  }
+
+  private async Task SendInviteEmailsAsync(List<Guid> userIds, string eventName)
+  {
+    List<UserData> usersData = await _userService.GetUsersDataAsync(userIds);
+
+    if (usersData is null || !usersData.Any())
+    {
+      return;
+    }
+
+    foreach (UserData user in usersData)
+    {
+      await _emailService.SendAsync(
+        user.Email,
+        "Invite to event",
+        $"You have been invited to event {eventName}");
+    }
   }
 
-  public async Task<OperationResultResponse<Guid?>> ExecuteAsync(CreateEventRequest request)
+  public async Task<OperationResultResponse<bool>> ExecuteAsync(CreateEventRequest request)
   {
     if (!await _accessValidator.HasRightsAsync(Rights.AddEditRemoveUsers))
     {
-      return _responseCreator.CreateFailureResponse<Guid?>(HttpStatusCode.Forbidden);
+      return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.Forbidden);
     }
 
     ValidationResult validationResult = await _validator.ValidateAsync(request);
     if (!validationResult.IsValid)
     {
-      return ResponseCreatorStatic.CreateResponse<Guid?>(
+      return _responseCreator.CreateFailureResponse<bool>(
         HttpStatusCode.BadRequest,
-        errors: validationResult.Errors.Select(er => er.ErrorMessage).ToList());
-    }
+        validationResult.Errors.Select(er => er.ErrorMessage).ToList());
+    };
 
     DbEvent dbEvent = _mapper.Map(request);
+    OperationResultResponse<bool> response = new();
+    response.Body = await _repository.CreateAsync(dbEvent);
```

Comment:
`OperationResultResponse<bool> response = new();
response.Body = await _repository.CreateAsync(_mapper.Map(request));`

Reply:
I use dbEvent in send emails method
left it as it was